#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: libndb.ver.in file uses NDB_SHARED_LIB_MAJOR_VERSION which isn't set by cmake

@DPATCH@
diff -urNad mysql-cluster-7.2.5.orig/cmake/mysql_version.cmake mysql-cluster-7.2.5/cmake/mysql_version.cmake
--- mysql-cluster-7.2.5.orig/cmake/mysql_version.cmake	2012-01-30 17:47:59.000000000 +0000
+++ mysql-cluster-7.2.5/cmake/mysql_version.cmake	2012-02-20 21:28:18.000000000 +0000
@@ -18,6 +18,7 @@
 #
 
 SET(SHARED_LIB_MAJOR_VERSION "18")
+SET(NDB_SHARED_LIB_MAJOR_VERSION "6")
 SET(PROTOCOL_VERSION "10")
 SET(DOT_FRM_VERSION "6")
 
